cmd/cronosd/opendb/opendb_rocksdb.go

```diff
@@ -33,8 +41,13 @@ func OpenDB(home string, backendType dbm.BackendType) (dbm.DB, error) {
 func OpenReadOnlyDB(home string, backendType dbm.BackendType) (dbm.DB, error) {
```

Comment:
could add an `openDB` with readonly flag to avoid change twice

Reply:
done